consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/headervalidationrules/BftValidatorsValidationRule.java

```diff
@@ -45,27 +54,45 @@ public boolean validate(
       final BftContext bftContext = context.getConsensusState(BftContext.class);
       final BftExtraData bftExtraData = bftContext.getBlockInterface().getExtraData(header);
 
-      final NavigableSet<Address> sortedReportedValidators =
-          new TreeSet<>(bftExtraData.getValidators());
+      // validators and vote may be empty if they are supplied by a contract.
```

Comment:
This comment is slightly misleading, the validators and vote must be empty.

Reply:
updated.